bot/menu.py

```diff
@@ -49,9 +49,23 @@ async def callback(self, interaction: Interaction):
             The action implemented by the user that needs to be notified.
             In the context of the bot, the action is a slash command
         """
-        await interaction.response.send_message(
-            f"Thank you for selecting {self.values}", ephemeral=True
-        )
+        response_message = ""
+        if self.is_adding_subscriber:
+            response_message = "You are now subscribed to "
+        else:
+            response_message = "You are now unsubscribed from "
+        converted_menu_values = set(map(int, self.values))
+        formatted_selected_tags = list()
+        for option in self.options:
+            if option.value in converted_menu_values:
+                if option.emoji:
+                    formatted_selected_tags.append(
+                        option.emoji.name + " " + option.label
+                    )
+                else:
+                    formatted_selected_tags.append(option.label)
+        response_message += ", ".join(formatted_selected_tags)
+        await interaction.response.send_message(f"{response_message}", ephemeral=True)

```

Comment:
out of curiosity do you know the difference between using something like 

```python
someString = f"{variable}"

# and

someString = "{}".format(variable)
```

?

I think it could be clearer too if we do something like 

```python
await interaction.response.send_message(f"You are now {action_type to/from} {', '.join(list_of_tags)}", ephemeral=True)
```

what do you think about this? I've never been a fan of string joins with + or += when i can avoid it, interpolation has always felt like a cleaner pattern

Reply:
I think this is possible, I am entirely new to this syntax coming from Java but I'll do some reading.

Reply:
7871ce7 - changes to interpolation pattern